### ISSUE TITLE
Shorcut page subpages leads to 404 page

### ISSUE THREAD

**Issue by ogrosko** [dmitryd/typo3-realurl#162]
In typo3 v7.6.6 and realurl v2.0.14 I have same problem like here #42 
But only on localized pages
When I loged off from BE it works

**Comment by dmitryd** [comment]
Should be solved @ 2.1.0

**Comment by Juerg-E** [comment]
Tested on Typo3 8.7.19 and realurl 2.4.0
Downgraded to 2.3.0 fixes my Bug.

```
TEST:
my tree:
->root
  -> 113
    -> 141
  ->  52
    ->  61
      -> 857

my "Shortcut Target":
default Language: id 52 -> 61 -> 141 [OK]
other Languages : id 52 -> 857 (page under 61) [wrong with realurl 2.4.0, OK with realurl 2.3.0]
```
